docs/Automation-Observability.md

```diff
@@ -67,12 +67,80 @@ Run the script to create the Diagnostic Setting.
+|Duration Selector | This selector allows a user to filter based on different time slices.  The default selector is set for 2 days. | [Figure 3](#time-range-selector) |[<img src="./assets/Automation-Dashboard/grafana-Timeline.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+| Number of Jobs Created | Number of automation jobs "created" within the time segment selected. | [Figure 4](#number-of-jobs-created) |[<img src="./assets/Automation-Dashboard/grafanaJobs-Created.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+| Number of Jobs Completed | Number of automation jobs "completed" within the time segment selected. | [Figure 5](#number-of-jobs-completed) |[<img src="./assets/Automation-Dashboard/grafanaJobs-Completed.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+| Number of Jobs with Errors | Number of automation jobs "with errors" within the time segment selected. | [Figure 6](#number-of-job-errors) |[<img src="./assets/Automation-Dashboard/grafanaJobs-Errors.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+| Time Series Diagram  | The time series diagramming the created/completed versus created and completed with errors. | [Figure 7](#time-series-diagram) |[<img src="./assets/Automation-Dashboard/grafana-TimeSeries.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+| Log List | The log list allows a user to see the errors that occured, as indicated in the element in Figure 6. | [Figure 8](#error-log-entries) | [<img src="./assets/Automation-Dashboard/grafana-Logs.png" width="250"/>](./assets/Automation-Dashboard/grafanaLogIn.png) |
+
 ### Troubleshooting with Grafana Dashboard
 
-- TODO: how to use grafana dashboard for troubleshooting
-  - what information is displayed in grafana dashboard?
-  - how to interpret it?
-  - when to take action?
+#### Time Range Selector
+
+![Time Selector](./assets/Automation-Dashboard/grafana-Timeline.png)
+Figure 3:  Grafana time line selection
+
+As show above a user can use the dropdown to adjust the timeline by which the data in the graph is loaded and thus a person can do analysis to ascertain the health of the automation process.  As shown in [Figure 3](#time-range-selector).
+
+#### Number of Jobs Created
+
+![Jobs Created](./assets/Automation-Dashboard/grafanaJobs-Created.png)
+Figure 4:  Grafana Jobs Created
+
+In [figure 4](#number-of-jobs-created) the number of jobs created to run during this time segment is listed.  This number is misleading, as it does not mean "new automation jobs" were created, but rather automation job instances created for running.  This is similar to the concept in American game baseball of "[On Deck](https://www.dictionary.com/browse/on--deck)".
```

Comment:
Should we change the title of this section of the dashboard? We could also include "automation job instances created for running" somewhere in the description tooltip in the dashboard.

Reply:
I like this suggestion, changing now.